### ISSUE TITLE
Creating servers fails

### ISSUE THREAD

**Issue by tlaundal** [SpaceCP/Bugs#98]
On Windows with Houston build 172 and DEV.

After going through the first dialog window for creating a server, I get the error below when I press `CREATE YOUR MINECRAFT SERVER NOW`.

```
info: bukkit: Loading as default vanilla Minecraft server
warn: bukkit: Failed to save config file.ENOENT: no such file or directory, open 'd:\Tobias\Downloads\houston\servers\bukkit\server.properties', file=server.properties, server=bukkit,  build=91b40b0be3a445b785e4b07c1d98dab06d5a1353_172
```

**Comment by JamyDev** [comment]
Well I think it's a bukkit specific issue

**Comment by tlaundal** [comment]
This is not bukkit specific, as it happens before I have selected a server jar

**Comment by JamyDev** [comment]
Oh, I was confused since you called your server bukkit :P

**Comment by tlaundal** [comment]
Yeah, I had a feeling that would cause confusion =)

**Comment by JamyDev** [comment]
Fixed by updating uniparse to 1.0.6

**Comment by tlaundal** [comment]
By running `npm i uniparse@1.0.6` and then `npm install` I was able to fix this on build 172.
